packages/network-controller/src/NetworkController.test.ts

```diff
@@ -3722,6 +3724,108 @@ describe('NetworkController', () => {
       });
     });
   });
+
+  describe('Network Configurations', () => {
```

Comment:
I've begun grouping tests by method rather than generic concept. This should help when cross-referencing a test file with its implementation file. Also it sets the subject for these tests ("What should add a network configuration? The `upsertNetworkConfiguration` method"). What do you think about using `describe('#upsertNetworkConfiguration', ...)`?

Reply:
Agreed, but on that naming suggestion, I'd rather we not adopt the convention of using a `#` before each method name. The `#` already denotes a private class member.

Reply:
Ah, that's a good point. That is probably a holdover I have from Ruby :)